backup_and_restore/application_backup_and_restore/features-and-plugins/oadp-features-plugins.adoc

```diff
@@ -0,0 +1,31 @@
+:_content-type: ASSEMBLY
+[id="oadp-features-plugins"]
+= OADP features and plugins
+include::_attributes/common-attributes.adoc[]
+:context: oadp-features-plugins
+
+toc::[]
+
+OpenShift API for Data Protection (OADP) features provide options for backing up and restoring applications.
+
+The default plugins enable Velero to integrate with certain cloud providers and to back up and restore {product-title} resources.
+
+
+
+////
+include::modules/oadp-features.adoc[leveloffset=+1]
+include::modules/oadp-plugins.adoc[leveloffset=+1]
+include::modules/oadp-configuring-velero-plugins.adoc[leveloffset=+1]
+
+[id="oadp-support-for-ibm-power-and-ibm-z"]
+== OADP support for IBM Power and {ibmzProductName}
+
+OpenShift API for Data Protection (OADP) is platform neutral. The information that follows relates only to IBM Power and to {ibmzProductName}.
+
+OADP 1.1.0 was tested successfully against {product-title} 4.11 for both IBM Power and {ibmzProductName}. The sections that follow give testing and support information for OADP 1.1.0 in terms of backup locations for these systems.
```

Comment:
Is this 1.2 or 1.1 documentation?
am i missing somthing?

Reply:
The docs cover 1.2 and 1.1. 

But I get your point, I need to create a *doc improvements* ticket to update the docs for this section for 1.2